stubs/gevent/gevent/_fileobjectcommon.pyi

```diff
@@ -0,0 +1,368 @@
+import io
+from _typeshed import (
+    FileDescriptorOrPath,
+    OpenBinaryMode,
+    OpenBinaryModeReading,
+    OpenBinaryModeUpdating,
+    OpenBinaryModeWriting,
+    OpenTextMode,
+    ReadableBuffer,
+)
+from types import TracebackType
+from typing import IO, Any, AnyStr, ClassVar, Generic, TypeVar, overload
+from typing_extensions import Literal, Self
+
+from gevent.lock import DummySemaphore, Semaphore
+from gevent.threadpool import ThreadPool
+
+_IOT = TypeVar("_IOT", io.TextIOWrapper, io.FileIO, io.BufferedRandom, io.BufferedWriter, io.BufferedReader, IO[bytes], IO[Any])
```

Comment:
Constrained TypeVars are always a bit suspicious. They make sense if something supports a fixed set of classes. But this one includes the abstract `IO` class, so I don't think that applies here. Should we instead use something like `bound=IO[Any]`?

Reply:
Would that actually cover all the types? But I can certainly try it. Otherwise I suppose I can add whatever isn't covered into a union and make that the bound.

I think initially the type var only contained the specific classes from `io`, since I thought those were the only possible cases, but I had to add `IO[bytes]` and `IO[Any]` later on to cover a bunch of corner cases. But changing it to a type var with an upper bound instead of adding more types to the constrained type var, would probably have made more sense at the time.

Reply:
Looks like it's fine